conf/broker.conf

```diff
@@ -55,6 +55,9 @@ advertisedAddress=
 # The Default value is absent, the broker uses the first listener as the internal listener.
 # internalListenerName=
 
+# Enable or disable the proxy protocol.
+proxyProtocolEnabled=false
```

Comment:
This naming can be confusing since we already have a "proxy" for Pulsar protocol

Reply:
How about changing it to `haProxyProtocolEnabled`?

Reply:
`haProxyProtocol` looks much better than `proxyProtocol`.

Reply:
Ok, sounds good.